i6_models/parts/lstm.py

```diff
@@ -0,0 +1,70 @@
+__all__ = ["LstmBlockV1Config", "LstmBlockV1"]
+
+from dataclasses import dataclass
+import torch
+from torch import nn
+from typing import Dict, Tuple, Union
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class LstmBlockV1Config(ModelConfiguration):
+    input_dim: int
+    hidden_dim: int
+    num_layers: int
+    bias: bool
+    dropout: float
+    bidirectional: bool
```

Comment:
should we allow this? I feel like if we have bidirectional here, the BLSTM part becomes redundant, which is maybe okay, but might also cause two different branches that do the same, which I am not sure we want (if there are potential extensions later). We could maybe also just deprecate the BLSTM block?